lib/puppet/indirector/json.rb

```diff
@@ -52,7 +52,7 @@ def load_json_from_file(file, key)
     json = nil
 
     begin
-      json = File.read(file)
+      json = File.read(file).force_encoding(Encoding::UTF_8)
```

Comment:
I can probably figure out where I did that and share it - it may have merit with fixing this issue.